html/semantics/forms/form-submission-0/resources/targetted-form.js

```diff
@@ -19,3 +19,10 @@ function submitPromise(form, iframe) {
     form.submit();
   });
 }
+
+function loadPromise(iframe) {
+  return new Promise((resolve, reject) => {
+    iframe.onload = () => resolve(iframe.contentWindow.location.search);
```

Comment:
Optional nit: it seems like this can just be `resolve()`; the value is not used anywhere.

Reply:
Oh, right.  Removed the argument.